land_claims.civmap.json

```diff
@@ -241,5 +241,13 @@
202],[4270,-6164],[4246,-6163],[4177,-6150],[4157,-6127],[4141,-6047],[4080,-5988],[4048,-5965],[3998,-5981],[3942,-5972],[3901,-5967],[3842,-5937],[3765,-5937],[3741,-5958],[3727,-6009],[3720,-6052],[3678,-6067],[3635,-6121],[3592,-6173],[3566,-6241],[3563,-6284],[3514,-6330],[3568,-6397],[3526,-6428],[3491,-6421],[3432,-6436],[3381,-6405],[3309,-6426],[3222,-6545],[3212,-6568],[3160,-6583],[3151,-6654],[3037,-6698],[2941,-6677],[2869,-6742],[2817,-6793],[2774,-6750],[2688,-6698],[2633,-6716],[2574,-6697],[2525,-6684],[2508,-6642],[2556,-6625],[2572,-6605],[2574,-6512],[2572,-6495],[2575,-6373],[2548,-6369],[2527,-6348],[2503,-6338],[2487,-6318],[2352,-6109],[2008,-5911],[1744,-5921],[1692,-5752],[1707,-5719],[1717,-5698],[1709,-5631],[1693,-5574],[1642,-5479],[1598,-5433],[1438,-5339],[1294,-5263],[1138,-5171],[990,-5079],[1079,-5044],[1311,-4800],[1376,-4764],[1492,-4853],[1534,-4831],[1596,-4852],[1734,-4811],[1810,-4851],[1925,-4665],[1941,-4645],[1946,-4596],[1954,-4538],[1891,-4474],[1855,-4425],[1800,-4427],[1734,-4354],[1737,-4208],[1679,-4134],[1709,-3949],[1855,-3716],[1868,-3540],[2220,-3380],[2559,-2891],[2659,-2879],[2719,-2890],[2763,-2856],[2816,-2886],[2868,-2900],[2974,-2900],[2999,-2860],[2966,-2810],[2951,-2724],[2840,-2789],[2742,-2685],[2713,-2613],[2677,-2596],[2678,-2489],[2700,-2457],[2757,-2440],[2820,-2456],[2848,-2448],[2882,-2373],[3051,-2331],[3080,-2333],[3118,-2373],[3141,-2380],[3170,-2356],[3212,-2345],[3221,-2311],[3243,-2291],[3260,-2256],[3279,-2250],[3305,-2261],[3323,-2271],[3339,-2272],[3396,-2223],[3413,-2181],[3427,-2165],[3445,-2151],[3454,-2138],[3467,-2134],[3499,-2150],[3529,-2150],[3567,-2125],[3573,-2113],[3566,-2096],[3568,-2081],[3583,-2066],[3601,-2071],[3627,-2097],[3634,-2101],[3642,-2125],[3658,-2125],[3658,-1637],[3627,-1577],[3832,-1515],[4106,-1312],[4154,-1052],[4364,-1008],[4591,-883],[4910,-1054],[4954,-1204],[4902,-1514],[5032,-1896],[5620,-2206],[5460,-3008],[5544,-3396],[5834,-3521],[5858,-3572],[5862,-3684],[5566,-3964],[5485,-4097],[5482,-4201],[5271,-4201],[5090,-4514],[4976,-4593],[4930,-4669],[4949,-4710],[4993,-4757],[5010,-4797],[5066,-4814],[5097,-4859],[5076,-4895],[4989,-4917],[4977,-4936],[4939,-4975],[4943,-5000],[4918,-5018],[4887,-5014],[4869,-5029],[4830,-5056],[4812,-5085],[4794,-5104],[4774,-5117],[4788,-5149],[4789,-5238],[4850,-5265],[4905,-5330],[4936,-5404],[4708,-5498],[4568,-5800],[4762,-6030],[4706,-6314],[4578,-6355],[4546,-6320]],[[5709,-5929],[5703,-5846],[5734,-5748],[5722,-5666],[5757,-5564],[5521,-5536],[5513,-5518],[5489,-5518],[5478,-5514],[5456,-5514],[5446,-5506],[5446,-5482],[5460,-5469],[5444,-5461],[5436,-5466],[5418,-5466],[5400,-5483],[5380,-5484],[5360,-5484],[5348,-5465],[5345,-5465],[5336,-5470],[5328,-5478],[5326,-5484],[5321,-5486],[5309,-5480],[5299,-5473],[5208,-5489],[5140,-5555],[5150,-5739],[5288,-5858],[5326,-5952],[5684,-6004]],[[6281,-5895],[6339,-5921],[6350,-5966],[6331,-6004],[6275,-6031],[6215,-6032],[6198,-5975],[6231,-5920]]]},
```

Comment:
pretty sure nyasaland already exists? also please sort the entries by name alphabetically

Reply:
Alright will update changes later